File: beecrowd/1_Beginner/1094_Experiments.c
#include <stdio.h>
#include <math.h>

int main()
{
    int C = 0;
    int R = 0;
    int S = 0;

    char type;
    int amount = 0, totalAnimal = 0;

    int myNum;
    scanf("%d", &myNum);
//    printf("%d",myNum);


    for(int i=0; i<myNum; i++)
    {
        scanf("%d %c",&amount, &type);
        totalAnimal = totalAnimal + amount;
        if(type == 'C')
        {
            C = C + amount;
        }
        else if(type == 'R')
        {
            R = R + amount;
        }
        else if(type == 'S')
        {
            S = S + amount;
        }

    }

    printf("Total: %d cobaias\n",totalAnimal);
    printf("Total de coelhos: %d\n",C);
    printf("Total de ratos: %d\n",R);
    printf("Total de sapos: %d\n",S);

    float avg = (C*100.00)/totalAnimal;
    printf("Percentual de coelhos: %.2f %%\n", avg);
    avg = (R*100.00)/totalAnimal;
    printf("Percentual de ratos: %.2f %%\n",avg);
    avg = (S*100.00)/totalAnimal;
    printf("Percentual de sapos: %.2f %%\n",avg);

    return 0;
}

File: beecrowd/1_Beginner/Python/1094_Experiments.py
N = int(input())
C = int(0)
R = int(0)
S = int(0)
totalAnimal = int(0)

for i in range(N):
    row = list(map(str, input().split()))
    amount = int(row[0])
    type = row[1]

    totalAnimal = totalAnimal + amount;

    if type == 'C':
        C = C + amount;
    elif type == 'R':
        R = R + amount;

    elif type == 'S':
        S = S + amount;

print("Total: "+ str(totalAnimal)+" cobaias");
print("Total de coelhos: "+ str(C));
print("Total de ratos: "+ str(R));
print("Total de sapos: "+ str(S));

avg = "{:.2f}".format((C*100.00)/totalAnimal)
print("Percentual de coelhos: " + str(avg) +" %")
avg = "{:.2f}".format((R*100.00)/totalAnimal)
print("Percentual de ratos: " + str(avg) +" %")
avg = "{:.2f}".format((S*100.00)/totalAnimal)
print("Percentual de sapos: " + str(avg) +" %")